packages/manager/modules/pci/src/projects/project/storages/snapshots/snapshot/create-volume/create-volume.module.js

```diff
@@ -19,8 +17,7 @@ angular
     'ovh-api-services',
     'ngTranslateAsyncLoader',
     'pascalprecht.translate',
-    blockStorages,
-    volumeEdit,
+    pciBlockStorage,
```

Comment:
not needed as a micro app is independent of other apps

Reply:
https://github.com/ovh/manager/pull/10563#discussion_r1472797974